app/views/shared/_sidebar.html.erb

```diff
@@ -13,7 +13,7 @@
     </span>
       <% if current_user.present? %>
         <span class="sidebar-item"><button id="modal-register" class="btn text-left">
-          <button id="modal-register" class="btn text-left"><%= link_to "Mi cuenta", user_path(current_user) %></button>
+          <button id="modal-register" class="btn text-left"><%= link_to "Mi cuenta", my_account_path(current_user) %></button>
```

Comment:
What happens if you do not pass the `current_user` as param here?
Try it 😄 
I think it's not needed

```suggestion
          <button id="modal-register" class="btn text-left"><%= link_to "Mi cuenta", my_account_path %></button>
```